msgbus.go

```diff
@@ -14,11 +14,25 @@ type MessageConfirm interface {
 }
 
 type PublishOption struct {
-	RoutingKey string
+	RoutingKey   string
+	ExchangeType string
```

Comment:
@aalexanderkevin I think better to remove this option, when the routing key is not empty it will only work if the exchanget type is direct. So no need to make the user set it himself. Wdyt?